README.md

```diff
@@ -119,11 +120,12 @@ import smartcar "github.com/smartcar/go-sdk"
 ## Pro Features
 
 ### Compatibility
-Compatibility allows you to verify if a particular VIN is compatible with a scope of permissions. This method should be used prior to directing a user to the Smartcar Connect flow. [Learn more on our doc center.](https://smartcar.com/docs/connect-pro/connect-compatibility/)
+Compatibility allows you to verify if a particular VIN is compatible with a scope of permissions. This method should be used prior to directing a user to the Smartcar Connect flow. [Learn more on our doc center.](https://smartcar.com/docs/connect-pro/connect-compatibility/). For details on how to specify country code strings refer to [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
```

Comment:
for the latter (country code description) should we just put that in the doc center? we need to specify somewhere that the only country supported atm is US

Reply:
definitely in api-reference, we should probably include that here too